kythe/typescript/indexer.ts

```diff
@@ -1031,6 +1031,15 @@ class Visitor {
     if (modulePath) {
       const kModule = this.newVName('module', modulePath);
       this.emitEdge(this.newAnchor(moduleRef), EdgeKind.REF_IMPORTS, kModule);
+    } else {
+      // Check if module being imported is declared via `declare module`
+      // somewhere and if so - output ref to that statement.
+      const decl = moduleSym.valueDeclaration;
+      if (decl.kind === ts.SyntaxKind.ModuleDeclaration) {
```

Comment:
if (ts.isModuleDeclaration(decl)) {
and then below you don't need the 'as' coercion anymore

Reply:
Done.